var/spack/repos/builtin/packages/liggghts/package.py

```diff
@@ -97,3 +113,14 @@ def edit(self, spec, prefix):
     def install(self, spec, prefix):
         mkdir(prefix.bin)
         install(os.path.join('src', 'lmp_auto'), prefix.bin.liggghts)
+
+        # Install libs
+        libs_in_spec = spec.variants['libs'].value
+        if libs_in_spec != None:
+            mkdir(prefix.lib)
+            if 'shared' in libs_in_spec:
```

Comment:
```suggestion
            if spec.satisfies("libs=shared"):
```
similarly for `static` down below